app/models/metric.rb

```diff
@@ -88,7 +91,11 @@ def name
   #
   # +type+:: Only :hits are supported right now.
   def default?(type)
-    system_name && system_name.to_sym == type.to_sym
+    if type == :hits
+      system_name.to_s =~ self.class.hits_extended_system_name_regex
```

Comment:
I wouldn't ask you to push only for this, but imo we should have an instance method called `hits?` or `extended_name_hits?` or something like that which does exactly this, so it would be more easily readable, and I see this is used somehow often (for example few lines below I see `parent.system_name =~ self.class.hits_extended_system_name_regex` which then would be `parent.THIS_NEW_METHOD?`